.github/workflows/runtime_commit_artifacts.yml

```diff
@@ -118,14 +118,14 @@ jobs:
         run: |
           BASE_FOLDER='compiled-rn/facebook-fbsource/xplat/js'
           mkdir -p ${BASE_FOLDER}/react-native-github/Libraries/Renderer/
-          mkdir -p ${BASE_FOLDER}/RKJSModules/vendor/react/{scheduler,react,react-is,react-test-renderer}/
+          mkdir -p ${BASE_FOLDER}/RKJSModules/vendor/react/{scheduler,react,react-dom,react-is,react-test-renderer}/
```

Comment:
This line change doesn't match the description. Is that a second intentional change or accidental?

Reply:
`react-dom` was added recently as a new directory in the rn sync but it had been omitted here. I'll update the pr summary